src/ui/box/multisize_slide.js

```diff
@@ -10,7 +10,7 @@ export default class Slider extends React.Component {
   }
 
   // eslint-disable-next-line react/no-deprecated
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` is deprecated in React 18 unless you prefix it with `UNSAFE_`. There is an upgrade path but thought it better to do this for now so as not to rock the boat too much, and come back to this at some point before GA.